$NetBSD: patch-external_progressmeter.h,v 1.1 2015/03/25 02:10:02 rodent Exp $

Including sys/termios.h on FreeBSD throws a warning and breaks the build.

--- external/progressmeter.h.orig	2015-06-16 21:59:14.457000000 +0000
+++ external/progressmeter.h
@@ -26,6 +26,10 @@
 
 #include "config.h"
 
+#if HAVE_NBCOMPAT_H
+#include <nbcompat.h>
+#endif
+
 #if defined(HAVE_SYS_TERMIOS_H) && !defined(__FreeBSD__)
 #include <sys/termios.h>
 #elif HAVE_TERMIOS_H
